brianlong/www.validators.app
# 180761036 fix carousel with turbolinks

#### What's this PR do?
- Fixes homepage carousel 

#### How should this be manually tested?
- Go to the homepage
- Go to some other page and then go back to homepage via link that loads page via turbolinks (eg. 'Go back to homepage' button on 'Cluster Stats') 
- Make sure carousel on homepage works
- Check the same on mobiles

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/180761036)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
